2021/03.md

```diff
@@ -84,7 +84,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   | 3     | 15m     | [Class `static` Initialization Blocks](https://github.com/tc39/proposal-class-static-block) Stage 3 (conditional) Update ([slides](https://1drv.ms/p/s!AjgWTO11Fk-TkfhG_gVnKlNwMT-MyA?e=owLLRf)) | Ron Buckton |
     |   | 2     | 20m     | [Error Cause](https://github.com/tc39/proposal-error-cause) for Stage 3 | Chengzhong Wu, Hemanth HM |
     |   | 2     | 30m     | [Collection Normalization](https://github.com/tc39/proposal-collection-normalization) move Set to follow on due to bikeshed, prep for Stage 3 | Bradley Farias |
-    |   | 2     | 45m     | [Temporal](https://github.com/tc39/proposal-temporal) for Stage 3 | TBD |
+    |   | 2     | 60m+60m | [Temporal](https://github.com/tc39/proposal-temporal) for Stage 3 | TBD |
```

Comment:
we don't generally consider it timeboxed at all if it's over an hour; perhaps it'd be better to move it to the "longer discussions" section?